docs/development.rst

```diff
@@ -273,4 +264,4 @@ If you want to run this in a production environment you will need to make severa
 .. _`Running WSL GUI Apps on Windows 10`: https://techcommunity.microsoft.com/t5/windows-dev-appconsult/running-wsl-gui-apps-on-windows-10/ba-p/1493242
 .. _`Example Algorithm Image`: https://hub.docker.com/r/grandchallenge/otsu
 .. _`Evalutils`: https://pypi.org/project/evalutils/
-.. _`Getting Started with Evalutils`: https://evalutils.readthedocs.io/en/latest/usage.html#getting-started
\ No newline at end of file
+.. _`Getting Started with Evalutils`: https://evalutils.readthedocs.io/en/latest/usage.html#getting-started
```

Comment:
Just curious - why does this show up as a changed line? The lines seem identical to me. Is there some weird whitespace thing going on here?

Reply:
No idea I'm afraid, edited in vim under wsl with the checkout on the Linux partition.